src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -396,6 +264,188 @@
             </local:SettingContainer>
         </StackPanel>
 
+        <!--  Grouping: Color  -->

```

Comment:
I'm minorly concerned concern about moving the whole color scheme part down to where Cursor is. I suspect that the thing people doing personalization want **most** is their color scheme.

Reply:
Honestly, moving it back up will fix this issue:
> [Focus issue] Clicking on the appearance page yeets focus to FontFace control

I think I'll reorganize the page a bit